src/mono/mono/metadata/gc-internals.h

```diff
@@ -170,6 +170,8 @@ typedef void (*MonoFinalizationProc)(gpointer, gpointer); // same as SGenFinaliz
 
 void  mono_gc_register_for_finalization (MonoObject *obj, MonoFinalizationProc user_data);
 void  mono_gc_add_memory_pressure (gint64 value);
+void  mono_gc_remove_memory_pressure (gint64 value);
```

Comment:
```suggestion
void  mono_gc_add_memory_pressure (guint64 value);
void  mono_gc_remove_memory_pressure (guint64 value);
```

Should be `guint64` in the impl too. Following the managed signature with `ulong`.